move-mutator/src/cli.rs

```diff
@@ -71,7 +84,7 @@ impl FromStr for ModuleFilter {
         match s {
             "all" => Ok(ModuleFilter::All),
             _ => Ok(ModuleFilter::Selected(
-                s.split(',').map(String::from).collect(),
+                s.split(&[';', ':', '-', ',']).map(String::from).collect(),
```

Comment:
I'm thinking about `:` symbol as it may be used to give a function with a module name like:
`module::func_test1`
I don't think we need to support functions with module names (there can even be an address name resulting with something like: `addr::module::func1`), but it will probably result in unintentional split values.